scRNA-seq/05-clustering_markers_scRNA.Rmd

```diff
@@ -0,0 +1,349 @@
+---
+title: "Clustering cells and finding marker genes scRNA-seq data"
+author: CCDL for ALSF
+date: 2021
+output:
+  html_notebook: 
+    toc: true
+    toc_float: true
+---
+
+## Objectives
+
+This notebook will demonstrate how to:
+
+- Annotate clusters of cells in single-cell data
```

Comment:
Annotate is pretty broad in that it has multiple meanings and I'm not sure what we're going to be doing. That might be fine, though.

Reply:
Yeah, will rephrase this to be more specific.